qancode.py

```diff
@@ -93,7 +93,7 @@
 --------
 Selenium webdriver for Chrome, Firefox.
 
-Safari Technology Preview version.
+fari Technology Preview version.
```

Comment:
Ooops, errant keystroke error. Thanks! I think breaking it up good idea, no need to postpone.